community/dbms/src/main/java/org/neo4j/server/configuration/ConfigLoader.java

```diff
@@ -61,6 +61,12 @@ public Config loadConfig( Optional<File> homeDir, Optional<File> configFile,
                 settingsClassesSupplier );
     }
 
+    public Config loadOfflineConfig( Optional<File> homeDir, Optional<File> configFile )
+    {
+        return loadOfflineConfig( homeDir, configFile,
+                Pair.of( GraphDatabaseSettings.auth_enabled.name(), Settings.FALSE ) );
+    }
+
```

Comment:
Where does this get called from? I'm worried that it's a bit unexpected to get auth disabled just because you don't pass any overrides.